frontend/packages/db-structure/src/parser/schemarb/parser.ts

```diff
@@ -278,6 +309,51 @@ function extractRelationshipTableNames(
   return ok([primaryTableName, foreignTableName])
 }
 
+function extractCheckConstraint(
```

Comment:
Here is a function to get check constraint from [`add_check_constraint`](https://api.rubyonrails.org/v8.0.2/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_check_constraint) statement.

It will extract the table name, detail and constraint name from the statement.